src/v3/src/components/AuthCoin/AuthCoinConfig.ts

```diff
@@ -112,6 +112,13 @@ const AuthCoinByAuthenticatorKey: Record<string, AuthCoinConfig> = {
     description: loc('factor.call', 'login'),
     iconClassName: 'mfa-okta-phone',
   },
+  [IDX_STEP.PIV_IDP]: {
```

Comment:
From my understanding, we need this to be different because the original `redirect-idp` remediation name is shared between PIV/CAC and custom IDPs. This was established in v2 as the work around for the PIV/CAC implementation to distinguish between the different types of `redirect-idp` remediations.

Reply:
Ohh, I think you mean just the actual constant being used (`AUTHENTICATOR_KEY` vs `IDX_STEP`). I don't _think_ it is an issue, originally, I didn't add PIV_IDP to the `AUTHENTICATOR_KEY` constant because I thought that would be confusing since it isn't actually an authenticator key.